android/app/src/main/res/layout/activity_parent_quest_details.xml

```diff
@@ -5,73 +5,80 @@
 
     <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        android:layout_height="wrap_content"
+        android:padding="25dp">
 
         <ImageView
             android:id="@+id/quest_image_view"
             android:layout_width="80dp"
             android:layout_height="80dp"
             android:layout_centerHorizontal="true"
-            android:layout_marginTop="20dp" />
+            android:layout_marginBottom="20dp"
+            android:src="@drawable/broom"/>
```

Comment:
why is `broom` hardcoded?

Reply:
just makes it easier to see without having to build forgot to remove it tho